src/Schedulers/OrdersScheduler.php

```diff
@@ -165,6 +165,12 @@ public static function import( $order_id ) {
 			CustomersDataStore::sync_order_customer( $order_id ),
 		);
 
+		$order_refunds = $order->get_refunds();
+
+		foreach ( $order_refunds as $refund ) {
```

Comment:
I think this loop is going to end up with some level of repeated imports.  I think this fine from a data standpoint, but will have some impact on performance.

Maybe this would even allow us to skip queueing refunds in the scheduler and simply defer to this loop to handle syncing refunds.  @jeffstieler any thoughts on this?

Reply:
Is this change required to resolve the issue?

I'm a bit hesitant to introduce this loop as it somewhat obscures the amount of work being done in each import (though it may not be much.. how many refunds can orders have?) but mostly because we already have something working.

Reply:
>how many refunds can orders have?

An order can technically have an infinite amount of refunds, though we know realistically the median number of refunds is `1`.

>Is this change required to resolve the issue?

Another option would be to watch for `woocommerce_order_status_changed` and schedule all child refunds on that hook.

Though looking at what we currently import on an order, I'm wondering if `sync_refunds` is not a good way to simplify the scope of these imports and our logic around an order import.

```php
			OrdersStatsDataStore::sync_order( $order_id ),
			ProductsDataStore::sync_order_products( $order_id ),
			CouponsDataStore::sync_order_coupons( $order_id ),
			TaxesDataStore::sync_order_taxes( $order_id ),
			CustomersDataStore::sync_order_customer( $order_id ),
```

Reply:
After discussing in Slack, I believe that the potential impact from this loop is likely far less than the existing sync logic (namely `sync_order_products`). Let's :shipit: